Kconfig

```diff
@@ -30,3 +30,9 @@ config GUESTAGENT_ARCH_RISCV64
 	help
 		Build lima-guestagent for "riscv64" Arch
 	default y
+
+config PLUGIN_VIRT
+	bool "plugin for vmType virt"
```

Comment:
I think this should be called "libvirt"

Reply:
I also wonder if we can split out this to a separate plugin binary.

Same applies to the vbox driver, cloud drivers, etc.

Reply:
Sure, it just requires more work. And it would be needed, for cloud.
* https://github.com/lima-vm/lima/discussions/2007